src/util/tubemap.js

```diff
@@ -967,6 +967,37 @@ function compareReadIncomingSegmentsByComingFrom(a, b) {
   ); // neither has y-property
 }
 
+// Compare tracks based on ordering at their first convergence
+function compareTrackByInitialOrdering(trackA, trackB) {
+  // Find the first node where the two tracks converge, sort by layer
+  if (!trackA.hasOwnProperty("path")) return -1;
+  if (!trackB.hasOwnProperty("path")) return 1;
+
+  const pathA = trackA.path;
+  const pathB = trackB.path;
+
+  let AIndex = 0;
+  let BIndex = 0;
+  while (AIndex < pathA.length && BIndex < pathB.length) {
+    let trackAOrder = pathA[AIndex].order;
+    let trackBOrder = pathB[BIndex].order;
+    if (trackAOrder === trackBOrder && pathA[AIndex].node && pathB[BIndex].node) {
+      return pathB[BIndex].y - pathA[AIndex].y;
+    } else if (trackAOrder < trackBOrder) {
+      AIndex += 1;
+    } else if (trackAOrder > trackBOrder) {
+      BIndex += 1;
+    } else {
+      // Orders are equal but are traversing out of nodes
+      AIndex += 1;
+      BIndex += 1;
+    }
+  }
```

Comment:
This looks like it might get slow for a lot of long tracks needing to be compared, but I can't think of a better way to do it that's simple.